packages/helpers/classes/mail.d.ts

```diff
@@ -119,7 +119,7 @@ export interface MailData {
   cc?: EmailData|EmailData[],
   bcc?: EmailData|EmailData[],
 
-  from: EmailData,
+  from?: EmailData,
```

Comment:
I'm not sure I like this change to the interface. This appears as if the `from` field is not required in a `MailData` payload, which is misleading.

Reply:
I see your point and I agree. I saw there is validations on creating the Mail message. This would be caught on execution, got excited and tried to solve all in one 😅